frontend/packages/dev-console/src/components/import/advanced/ResourceLimitSection.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import FormSection from '../section/FormSection';
+import ResourceLimitField from '../../formik-fields/ResourceLimitField';
```

Comment:
After adding the `ResourceLimitField` to index.ts, import it directly from  `formik-fields`